imagorpath/parse.go

```diff
@@ -136,6 +137,14 @@ func Apply(p Params, path string) Params {
 		filters, img := parseFilters(match[index])
 		p.Filters = append(p.Filters, filters...)
 		if img != "" {
+			if strings.HasPrefix(img, "b64:") {
+				// if image URL starts with b64: prefix, Base64 decode it according to "base64url" in RFC 4648 (Section 5).
+				result := make([]byte, base64.RawURLEncoding.DecodedLen(len(img[4:])))
+				// no way to return a possible encoding error here
```

Comment:
Use the original image path if decoding failed, is ok - in case if image actually name "b64:foo.jpg" - super uncommon but still possible